guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+
+- **Address** – The destination of the payment
+- **Amount** – How much is being requested (optional)
+- **From** – Name of who is sending the payment (optional)
+- **Purpose/Message** – Note with a description of what the payment is for (optional)
+
+While the application will generate a new address in the background, other information like; how much is being requested, the name of the sender, and what is being paid for has to be manually entered by the receiver. The order in which these properties get entered is up to you, and besides the address, all else is optional.
+
+<!--
+Technical nitpick;
+Addresses are not exactly "generated"
+since they are deterministic technically
+what happens is an unused address just gets chosen
+-->
+
+The more time passes, the more difficult it will be to recall the details of transactions. When transacting in Bitcoin there is no requirement for senders and receivers even to identify themselves, so both sides may end up having a transaction history with only amounts and random-looking text (addresses) to reference when trying to understand where some funds came from.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-amount.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-amount@2x.png"
+   alt-text = "Screen asking the receiver to enter the amount being requested"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-from.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-from@2x.png"
+   alt-text = "Screen asking the receiver to enter the name of the sender"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+</div>
+
+If your design’s goal is to get the receiver in the quickest time to share an address, you can choose to show them a new address straight away. The other properties can be left as optional fields. Consider the trade-off when optimizing this way, as there will be less useful contextual details for reports that you may want to provide in the future.
+
+<div class="image-slide-gallery">
```

Comment:
No need for the slide gallery if you'll have only one image. You can just use the picture include with "layout=background". Could also go for the split-column layout.

Reply:
Nice, the -background class needs to have a text-align: center though, shall I include that in this PR?

<img width="677" alt="Screenshot 2021-04-29 at 13 53 58" src="https://user-images.githubusercontent.com/183140/116546804-5b2a8f80-a8f2-11eb-8d72-4dcbf0f214ab.png">

<img width="812" alt="Screenshot 2021-04-29 at 13 53 07" src="https://user-images.githubusercontent.com/183140/116546719-3d5d2a80-a8f2-11eb-8d0a-eeb43feb7fa4.png">

Reply:
hmm - i think i mistook picture.html for image.html

Reply:
No. Conor is already adding this in #225. That's the reason why we need to separate content commits and code commits.

Reply:
been having issues with this picture/image component, there are no captions. im leaving it as a gallery for now - unless you can walk me through how to get it working as you have in mind.